build_and_install_habitat_sim_js.sh

```diff
@@ -0,0 +1,6 @@
+./build_js.sh --bullet
```

Comment:
This shouldn't be in the habitat-sim root folder. It is part of the example app so it should probably be at `examples/web_apps/webxr_hand_demo/`.